template/resources/leiningen/new/friboo/core.clj

```diff
@@ -2,14 +2,28 @@
   (:require [org.zalando.stups.friboo.config :as config]
             [org.zalando.stups.friboo.system :as system]
             [org.zalando.stups.friboo.system.http :as http]
-            [org.zalando.stups.friboo.log :as log]
+            [clojure.string :as str]
             [com.stuartsierra.component :as component]
+            [taoensso.timbre :as log]
+            [taoensso.timbre.appenders.core :as core-appenders]
             [{{namespace}}.api :as api])
   (:gen-class))
 
 (def default-http-config
   {:http-port 8080})
 
+(defn default-log-output-fn
+  ([data] (default-log-output-fn nil data))
+  ([_ data] (let [{:keys [level ?ns-str ?msg-fmt vargs ?err]} data]
+              (format "%5s [%s] %s - %s%s\n"
+                      (str/upper-case (name level))
+                      (.getName (Thread/currentThread))
+                      (str ?ns-str)
+                      (if-let [fmt ?msg-fmt]
+                        (apply format fmt vargs)
+                        (apply str vargs))
+                      (str (when ?err (log/stacktrace ?err)))))))
```

Comment:
Definitely had some hesitations about the way of configuring logging output here, tried to keep the format look like it was in the main library before and make the template work out of the box.